docs/config/README.md

```diff
@@ -143,13 +143,14 @@ This is the list of prompt-wide configuration options.
 
 ### Options
 
-| Option            | Default                         | Description                                                      |
-| ----------------- | ------------------------------- | ---------------------------------------------------------------- |
-| `format`          | [link](#default-prompt-format)  | Configure the format of the prompt.                              |
-| `right_format`    | `""`                            | See [Enable Right Prompt](/advanced-config/#enable-right-prompt) |
-| `scan_timeout`    | `30`                            | Timeout for starship to scan files (in milliseconds).            |
-| `command_timeout` | `500`                           | Timeout for commands executed by starship (in milliseconds).     |
-| `add_newline`     | `true`                          | Inserts blank line between shell prompts.                        |
+| Option                | Default                         | Description                                                      |
+| --------------------- | ------------------------------- | ---------------------------------------------------------------- |
+| `format`              | [link](#default-prompt-format)  | Configure the format of the prompt.                              |
+| `right_format`        | `""`                            | See [Enable Right Prompt](/advanced-config/#enable-right-prompt) |
+| `continuation_format` | `"$character"`                  | See [Continuation Prompt](/advanced-config/#continuation-prompt) |
```

Comment:
I'm a little concerned about the presence of this here, since it seems to imply that we can set the continuation prompt to something that will be dynamically updated with every prompt draw (like the right prompt is). Any ideas on how to clarify this? Perhaps a warning in advanced-config?

Reply:
I'm actually wondering if the static continuation prompt should utilize modules at all. Most modules seem to depend on the fact that they get rerun after every command, so they would be mostly useless here. Please let me know your opinion on this.

Do you think it would be a good idea to call it `continuation_prompt` instead of `continuation_format`; implying that it does not get formatted like the default and right prompts? If we choose to not utilize modules, I can also add this fact as a warning in advanced-config.